wrappers/ortools.rb

```diff
@@ -582,6 +582,7 @@ def run_ortools(problem, vrp, thread_proc = nil, &block)
       stdin&.close
       stdout_and_stderr&.close
       pipe&.close
+      thread_proc&.call([])
```

Comment:
What does this do exactly?

Reply:
Once the resolution is finished, the pid is not reseted. So, a DELETE request try to kill this pid which no more exist.